authentication/http.go

```diff
@@ -39,23 +37,25 @@ func WithTenant(next http.Handler) http.Handler {
 
 // WithTenantID returns a middleware that finds the tenantID using the tenant
 // from the URL parameters and adds it to the request context.
-func WithTenantID(tenantIDs map[string]string) Middleware {
+func WithTenantID(tenantName, tenantID string) Middleware {
```

Comment:
We need the tenant map because this maps from the tenant name to the tenant ID. The URL will only have the name as part of the HTTP path

Reply:
Now `withTenantID()` is called for each tenant == separate handler() for each tenant

Reply:
Got it👍